### PR TITLE
Add SidebarBottom to toggle sidebar filtering based on story status

### PR DESCRIPTION
**Pull request by ghengeveld**
Fixes AP-3301, AP-3302, AP-3303

Also removes the "pending" story status, because it shows as an ugly "clock" icon and we don't _really_ need to show the "in progress" state of a test in the sidebar.

Note: this depends on a canary version of Storybook and I had to downgrade to React 16 for Storybook compatibility.

Screenshot 2023-08-23 at 16 09 59

<details>
  <summary>📦 Published PR as canary version: <code>0.0.40--canary.45.460fa88.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @chromaui/addon-visual-tests@0.0.40--canary.45.460fa88.0
  # or 
  yarn add @chromaui/addon-visual-tests@0.0.40--canary.45.460fa88.0
  ```
</details>

### CODE REVIEW

**Review by weeksling — APPROVED** [review]
lgtm. Question regarding the new fixed version of storybook, and whether this should wait until that's stable to be merged

**Comment by weeksling** [comment]
@ghengeveld I approved from the look of the code, however testing this I couldn't actually see the filter menu in storybook. Is there anything extra you need to do to enable or, or should switching to this branch, running `yarn` and running `yarn start` show the new slot?

File: src/utils/testsToStatusUpdate.ts
Comment (tmeasday) [reply src/utils/testsToStatusUpdate.ts:10]: Agreed, I thought that we were going to show a grey pulsing dot on in progress tests.

File: package.json
Comment (tmeasday) [reply package.json:108]: Storybook is planning up upgrading to `react@18` as part of the 8.0 release at the end of the year.

File: src/SidebarBottom.tsx
Comment (tmeasday) [inline src/SidebarBottom.tsx:34]: What about errors?

**Review by tmeasday — APPROVED** [review]
Code LGTM but we should work out what we are doing with errors. I'm OK with creating a ticket for that and merging as is though.

File: src/components/SuffixInput.tsx
Comment (ghengeveld) [reply src/components/SuffixInput.tsx:66]: Nope, it's a React 16 thing.

File: src/utils/testsToStatusUpdate.ts
Comment (ghengeveld) [reply src/utils/testsToStatusUpdate.ts:10]: I would love to but not if it looks that awful.
@ndelangen can you fix the "pending" icon to "look pretty"?

File: src/SidebarBottom.tsx
Comment (ghengeveld) [reply src/SidebarBottom.tsx:34]: That doesn't seem to be a thing.

Screenshot 2023-08-24 at 14 57 33

File: src/utils/testsToStatusUpdate.ts
Comment (weeksling) [reply src/utils/testsToStatusUpdate.ts:10]: I think @MichaelArestad is going to be making some improvements to ui including those icons

Comment (MichaelArestad) [reply src/utils/testsToStatusUpdate.ts:10]: I am planning on working with @ndelangen on that.

File: package.json
Comment (ndelangen) [reply package.json:72]: I'm releasing an alpha version now:
https://github.com/storybookjs/storybook/pull/23926

**Pull request merged**
